src/test/static/project-with-errors/build/output.html

```diff
@@ -0,0 +1 @@
+<link rel="import" href="built-nonexistant-import.html">
```

Comment:
Nevermind I totes understand now.  I didn't get the naming `built-nonexistant-import.html` at first.